packages/core/src/types.ts

```diff
@@ -285,6 +286,16 @@ export class Config {
      * @type {string|undefined}
      */
     this.userAgent = undefined
+
+    /**
+     * Specify if telemetry collection is disabled.
+     *
+     * Metrics are only collected when enabled both in server and driver instances.
```

Comment:
I suggest to be more explicit in the API docs. What is being collected?

For inspiration https://github.com/neo4j/neo4j-python-driver/pull/965/files#diff-b83b53d2e790a152ae629a3756c2232706f9a9660ed385196b17f63362df2c87R668-R689